templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -13,23 +13,89 @@
         {block name='product_miniature_image'}
           <div class="{$componentName}__image-container thumbnail-container">
             {if $product.cover}
-              <img
-                src="{$product.cover.bySize.home_default.url}"
-                alt="{if !empty($product.cover.legend)}{$product.cover.legend}{else}{$product.name}{/if}"
-                loading="lazy"
-                data-full-size-image-url="{$product.cover.bySize.home_default.url}"
-                width="{$product.cover.bySize.home_default.width}"
-                height="{$product.cover.bySize.home_default.height}"
-                class="{$componentName}__image card-img-top"
-              />
+              
+                {if isset($product.cover.bySize.default_320.sources.avif)}
+                  <source 
+                    srcset="
+                      {$product.cover.bySize.default_120.sources.avif} 120w,
+                      {$product.cover.bySize.default_200.sources.avif} 200w,
+                      {$product.cover.bySize.default_320.sources.avif} 320w,
+                      {$product.cover.bySize.default_720.sources.avif} 720w"
+                    sizes="(min-width: 1300px) 720px, (min-width: 768px) 50vw, 100vw" 
+                    type="image/avif"
+                  >
+                {/if}
+
+                {if isset($product.cover.bySize.default_320.sources.webp)}
+                  <source 
+                    srcset="
+                      {$product.cover.bySize.default_120.sources.webp} 120w,
+                      {$product.cover.bySize.default_200.sources.webp} 200w,
+                      {$product.cover.bySize.default_320.sources.webp} 320w,
+                      {$product.cover.bySize.default_720.sources.webp} 720w"
+                    sizes="(min-width: 1300px) 320px, (min-width: 768px) 120px, 100vw" 
+                    type="image/webp"
+                  >
+                {/if}
+
+                <img
+                  class="{$componentName}__image card-img-top"
+                  srcset="
+                    {$product.cover.bySize.default_120.url} 120w,
+                    {$product.cover.bySize.default_200.url} 200w,
+                    {$product.cover.bySize.default_320.url} 320w,
+                    {$product.cover.bySize.default_720.url} 720w"
+                  sizes="(min-width: 1300px) 320px, (min-width: 768px) 120px, 100vw" 
+                  src="{$product.cover.bySize.default_720.sources.jpg}" 
```

Comment:
Let's use 320 as a fallback dimensions. The same for width and height attributes.